modules/docs/src/Volo.Docs.Admin.Web/DocsAdminWebModule.cs

```diff
@@ -27,6 +32,8 @@ public override void PreConfigureServices(ServiceConfigurationContext context)
             {
                 mvcBuilder.AddApplicationPartIfNotExists(typeof(DocsAdminWebModule).Assembly);
             });
+
+            context.Services.AddSignalR();
```

Comment:
This is already done in [AbpAspNetCoreSignalRModule](https://github.com/abpframework/abp/blob/dev/framework/src/Volo.Abp.AspNetCore.SignalR/Volo/Abp/AspNetCore/SignalR/AbpAspNetCoreSignalRModule.cs#L35).